src/plugins/intel_npu/src/backend/include/zero_backend.hpp

```diff
@@ -23,16 +23,15 @@ class ZeroEngineBackend final : public IEngineBackend {
     }
     const std::vector<std::string> getDeviceNames() const override;
     uint32_t getDriverVersion() const override;
-    uint32_t getDriverExtVersion() const override;
+    uint32_t getGraphExtVersion() const override;
 
     bool isBatchingSupported() const override;
     bool isCommandQueueExtSupported() const override;
 
     void* getContext() const override;
     void* getDriverHandle() const;
     void* getDeviceHandle() const;
-    char* getGraphExtName();
-    ze_graph_dditable_ext_last_t* getGraphDDITableExt();
+    const std::unique_ptr<ze_graph_dditable_ext_decorator>& getGraphDdiTable();
```

Comment:
I think having a reference for an unique pointer is too much. Can you please consider using a plain refence or keep a raw pointer?

Unique pointer does not seem to bring any benefit here, if I am missing anything.

Reply:
Changed to share just a raw pointer through methods.

Reply:
Keeping away from dangling pointers might as well be considered of low significance.